### PR TITLE
Added changelog

### PR DESCRIPTION
**Pull request by Naatan**
https://www.pivotaltracker.com/story/show/176649436

### CODE REVIEW

File: internal/runners/activate/changelog.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## Unreleased
+
+Available on channel `master`
+
+### Added
```
Comment (daved) [inline internal/runners/activate/changelog.md:12]: It is shown that each group of lists (added/changed/etc.) belong contained by another header that indicates the version and date:
```
## [1.0.0] - 2017-06-20
### Added
```

**Review by daved — CHANGES_REQUESTED** [review]

File: internal/runners/activate/changelog.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Changelog
```
[the remark this answers is not in the document]
Comment (daved) [reply internal/runners/activate/changelog.md:1]: This is regarding the file location, not the first line itself.

File: internal/runners/activate/changelog.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
[the remark this answers is not in the document]
Comment (Naatan) [reply internal/runners/activate/changelog.md:6]: Our `--version` output appends `-SHAxxx`, but we don't include that in the changelog here. The changelog is semver compatible. And I'd argue even the `--version` output is fairly compatible.

File: internal/runners/activate/changelog.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Naatan) [reply internal/runners/activate/changelog.md:1]: Woops! I thought I did put it in the root.

File: internal/runners/activate/changelog.md (hunk 1 above)
Comment (Naatan) [reply internal/runners/activate/changelog.md:12]: Correct

**Review by daved — APPROVED** [review]

**Pull request merged**